tracex_project/trace_comparator/templates/testing_comparison.html

```diff
@@ -11,38 +11,38 @@
 </head>
 <body class="main_body">
     <div class="container">
-    <h1>Extraction Result</h1>
+        <h1>Extraction Result</h1>
 
-    <h2>Patient Journey: {{ patient_journey_name }}</h2>
-    <p>{{ patient_journey }}</p>
-    <p><b>This is the event log extracted from the journey above</b></p>
-    <div id="xesContent" class="xes_container">
-        {{ pipeline_output|safe }}
-      </div>
+        <h2>Patient Journey: {{ patient_journey_name }}</h2>
+        <p>{{ patient_journey }}</p>
+        <p><b>This is the event log extracted from the journey above</b></p>
+        <div id="xesContent" class="xes_container">
+            {{ pipeline_output|safe }}
+        </div>
 
-    <div class="spinner-container" id="loading-container">
-        <span id="loading-spinner" class="loader not_visible"></span>
-    </div>
+        <div class="spinner-container" id="loading-container">
+            <span id="loading-spinner" class="loader not_visible"></span>
+        </div>
 
-    <div id="progress_box" class="not_visible">
-        <div class="progress-container">
-            <!-- Progress bar and message will be added here -->
+        <div id="progress_box" class="not_visible">
+            <div class="progress-container">
+                <!-- Progress bar and message will be added here -->
+            </div>
         </div>
-    </div>
 
 
-    <form method="post" >
-        {% csrf_token %}
-        <button type="submit" class="menu_button" id="progress_button">Start Comparing Against Ground Truth</button>
-    </form>
+        <form method="post" >
+            {% csrf_token %}
+            <button type="submit" class="menu_button" id="progress_button">Start Comparing Against Ground Truth</button>
+        </form>
```

Comment:
```suggestion
        <form method="post" >
            {% csrf_token %}
            <button type="submit" class="menu_button" id="progress_button">Start Comparing Against Ground Truth</button>
        </form>
        <script>
            function disableButton() {
                document.getElementById("progress_button").classList.add("processing");
            }
        </script>
```

Reply:
This will make the button unclickable after initial start of the comparison process